serverlessworkflow/modules/ROOT/pages/eventing/consume-producing-events-with-kafka.adoc

```diff
@@ -33,53 +26,59 @@ If using Maven, this means including following snippet in pom.xml:
+** `value.serializer`: This property is used for outgoing channels. Unless you have specific marshaling requirements, you must set this property to `org.apache.kafka.common.serialization.ByteArraySerializer` or `org.apache.kafka.common.serialization.StringSerializer`.
 
-* `connector`, which has to be set to `smallrye-kafka`
-* Depending on the channel being input or output: 
-** if incoming, `value.deserializer`. Unless you have specific marshaling needs, you should set it to `org.apache.kafka.common.serialization.ByteArrayDeserializer` or `org.apache.kafka.common.serialization.StringDeserializer`
-** if outgoing, `value.serializer`. Unless you have specific marshaling needs, you should set it to `org.apache.kafka.common.serialization.ByteArraySerializer` or `org.apache.kafka.common.serialization.StringSerializer`
+[TIP]
+====
+`topic` is another relevant but optional property that needs to be set for a channel. The `topic` property must contain the Apache Kafka topic name to be used for a channel. If `topic` property is not set, then the channel name is used as topic name. 
+====
 
-Another relevant, although optional, property is the `topic`. If present, it should contain the Kafka topic name to be used for that channel. If missing, the channel name will be used as topic name. 
+For more information about the properties that you can use for a channel when using Apache Kafka connector, see the link:{kafka_connector_properties_url}[properties] list.
 
-=== Channel Mapping Examples
+[[ref-example-smallrye-channel-mapping]]
+=== Examples of Smallrye channel mapping
 
-Let's illustrate previous explanation by reviewing the properties configured in several examples. 
+This section describes the examples of channel mapping by reviewing the properties that are configured in the previous section. 
 
-==== Channel per event type 
+One Smallrye channel per CloudEvent type::
++
+--
+The link:{callback_example_url}[`serverless-workflow-callback-quarkus`] example application uses two CloudEvent types, including `wait` (incoming) and `resume` (outgoing).
 
-link:{callback_example_url}[Callback example] uses two Cloud Event types: `wait`, which is incoming, and `resume`, which is outgoing. 
-
-Kafka topic names matches the cloud event types, therefore it makes sense to chose one channel per Cloud Event type mapping strategy. This implies two channels must be configured. Since the channel name matches the cloud event type, we do not need to indicate the optional `topic` property.
+As mentioned before, Kafka topic names matches the CloudEvent types. Therefore, it is suitable to select the mapping strategy of one channel per CloudEvent type. This suggests that two channels must be configured. Note that you do not need to set the optional `topic` property as the channel name matches the CloudEvent type.
```

Comment:
We did not mention anything before ;)
The sentence shoud be "Kafka topic names matches the cloudevent types", we are stating a fact that has a consequence in the next sentence

Reply:
Done, removed "As mentioned before"